meteor/server/api/ingest/syncChangesToPartInstance.ts

```diff
@@ -94,6 +94,7 @@ export function syncChangesToPartInstances(
 				const proposedPieceInstances = getPieceInstancesForPart(
 					cache,
 					previousPartInstance,
+					rundown as Rundown,
```

Comment:
please propogate the `ReadonlyDeep<Rundown>` through the other methods instead of casting here.
So far I've been lazily changing the typings as needed, so expect to need to change places